src/Compilers/CSharp/Portable/Symbols/TypeSymbol.cs

```diff
@@ -920,6 +943,111 @@ private static Symbol ComputeImplementationForInterfaceMember(Symbol interfaceMe
             return implicitImpl;
         }
 
+        private static Symbol FindMostSpecificImplementation(Symbol interfaceMember, TypeSymbol implementingType,
+                                                             ref HashSet<DiagnosticInfo> useSiteDiagnostics,
+                                                             DiagnosticBag diagnostics)
+        {
+            Symbol implicitImpl = FindMostSpecificImplementation(interfaceMember,
+                                                          implementingType,
+                                                          ref useSiteDiagnostics, out Symbol conflict1, out Symbol conflict2);
+
+            if ((object)conflict1 != null)
+            {
+                Debug.Assert(((object)implicitImpl == null));
+                Debug.Assert(((object)conflict1 != null));
```

Comment:
Or maybe you meant `conflict2 != null`.

---
In reply to: [119775932](https://github.com/dotnet/roslyn/pull/19876#discussion_r119775932) [](ancestors = 119775932)